packages/neo4j-driver-lite/README.md

```diff
@@ -1,7 +1,10 @@
-# Neo4j Driver for JavaScript
+# Neo4j Driver for JavaScript (lite)
```

Comment:
I am unsure about the placement of `(lite)`: as it is, it looks like it refers to Javascript. Maybe `Neo4j Lite Driver for JavaScript`, or `Neo4j Driver Lite for JavaScript`?

Reply:
I've update with the `Neo4j lite driver` name